### ISSUE TITLE
enable_cell_picking miss left_clicking keyword

### ISSUE THREAD

**Issue by zhang9song** [pyvista/pyvista#4381]
### Describe the bug, what's wrong, and what you expected.

Hi, 
In pyvista doc,  enable_cell_picking example is:

```py
import pyvista as pv
mesh = pv.Sphere(center=(1, 0, 0))
cube = pv.Cube()
pl = pv.Plotter()
_ = pl.add_mesh(mesh)
_ = pl.add_mesh(cube)
_ = pl.enable_cell_picking(left_clicking=True)
```
When I run it, I got error: TypeError: "left_clicking" is an invalid keyword argument for '_common_arg_parser'
I check source code, no left_clicking in enable_cell_picking function.

### Steps to reproduce the bug.

```py
import pyvista as pv
mesh = pv.Sphere(center=(1, 0, 0))
cube = pv.Cube()
pl = pv.Plotter()
_ = pl.add_mesh(mesh)
_ = pl.add_mesh(cube)
_ = pl.enable_cell_picking(left_clicking=True)
```

### System Information

```shell
TypeError: "left_clicking" is an invalid keyword argument for '_common_arg_parser'
```

**Comment by banesullivan** [comment]
Is fixed by #4254

**Comment by banesullivan** [comment]
To clarify, this is addressed in #4254 but maybe not what expected. After those changes, `enable_cell_picking()` is specifically for selection are based picking and it doesn't support clicking a single location. So `left_clicking` still isn't an option.

But there is a new approach for single cell picking with `enable_element_picking`:

```py
import pyvista as pv
from pyvista.plotting.opts import ElementType, PickerType
from pyvista import examples

mesh = examples.load_random_hills()

p = pv.Plotter()
p.add_mesh(mesh, show_edges=True, pickable=True)
p.enable_element_picking(mode=ElementType.CELL, line_width=10, left_clicking=True)
p.show()
```
